WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
```

Comment:
It's not a matter of religion alone: it's a matter of something unkind being done to members of the realm. A kind ruler would want to undo that.

As for why I say "Dark Ages", it's simple: the Dark Ages were dark because the "light" of the Roman Empire was extinguished, not because they were a time of ignorance or repression. The successors didn't have the same degree of "light".